### ISSUE TITLE
Add Hurl File JSON export 

### ISSUE THREAD

**Issue by jcamiel** [Orange-OpenSource/hurl#65]
Add an option (with `hurlfmt` ?) to export an Hurl file (the last) to json, to provide an agnostic file format.

**Comment by fabricereix** [comment]
### Response version and status

In the hurl file, version and status code asserts can be ignored with the wildcard characters.
In terms of json export, they could be simply removed from the object without ambiguities.
this would enable to use a number json type instead of string.

for example
```
"response": { "version": 1.0, "status": 200 }
"response": { "status": 200 }
```

rather than
```
"response": { "version": "1.0", "status": "200" }
"response": { "version": "*", "status": "200"}
```

**Comment by fabricereix** [comment]
implemented in #70 and #78
